core/rootcling_stage1/CMakeLists.txt

```diff
@@ -17,7 +17,9 @@ else()
   endif()
 endif()
 
-if(NOT builtin_clang)
+if(builtin_clang)
```

Comment:
Moving from the other PR which was closed: This condition is intended to pin clingMetaProcessor as a library when we use builtin_clang=Off. That is, when clang is externally prebuilt and we do not really need to track dependencies. I believe this change will lead to missing dependencies and we will start seeing compilation errors. Did you test that code with compiling ROOT with -Dbuiltin_clang=Off?